### PR TITLE
[Test] Test root-squash functionality with volume remount

### PR DESCRIPTION
**Pull request by msainiRedhat**
Signed-off-by: Manisha Saini <msaini@msaini.blr.csb>

### CODE REVIEW

File: tests/functional/nfs_ganesha/root-squash/test_nfs_ganesha_root_squash_mount_unmount.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+        if not ret:
+            raise ExecutionError("Failed:Volume %s is not unexported."
+                                 % self.volname)
+        g.log.info("Unexporting of volume is successful")
+
+        # Unmount and cleanup Volume
+        ret = self.unmount_volume_and_cleanup_volume(self.mounts)
+        if ret:
+            g.log.info("Successful unmount and cleanup of volume")
+        else:
```
Comment (kshithijiyer) [inline tests/functional/nfs_ganesha/root-squash/test_nfs_ganesha_root_squash_mount_unmount.py:218]: Do we really need this or can we do it similar to what we did in the above case?

**Review by kshithijiyer — COMMENTED** [review]
A minor comment rest looks good, please look into the build system failures.

**Review by kshithijiyer — APPROVED** [review]
LGTM

**Review by pranavprakash20 — APPROVED** [review]
LGTM.

**Pull request merged**